apigentools/config.py

```diff
@@ -36,49 +36,51 @@ def get_language_config(self, lang):
 
     def get_validation_commands(self):
         cmd_objects = []
+        fake_language = LanguageConfig(None, {}, self)
         for cmd in self.validation_commands:
-            cmd_objects.append(ConfigCommand("validation", cmd))
+            cmd_objects.append(ConfigCommand(None, "validation", cmd, fake_language))
         return cmd_objects
 
     @classmethod
     def from_file(cls, fpath):
         with open(fpath) as f:
-            config = json.load(f)
+            config = yaml.safe_load(f)
```

Comment:
IMHO loading based on suffix

Reply:
doesn't really matter, yaml will load json as well

Reply:
I would actually love to move to Yaml fully to be able to warn users of old config.json that they should migrate. I'm not generally opposed to supporting both, but the dillema of "I see both config.yaml and config.json, so how do I tell user *visibly enough* that I can only use one" I'd like to avoid.